<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Entry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search Box</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>731</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-20</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zoo Shop Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search By :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>120</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>189</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tables:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboTables"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable tables</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>187</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tables</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>140</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Become a Golden client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show goods for my pets</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Entry"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Developer"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_Entry"/>
    <addaction name="actionRemove_Entry"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Entry"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_Entry"/>
   <addaction name="separator"/>
   <addaction name="actionRemove_Entry"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Entry">
   <property name="icon">
    <iconset>
     <normaloff>:Icons/Misc-New-Database-icon.png</normaloff>:Icons/Misc-New-Database-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionEdit_Entry">
   <property name="icon">
    <iconset>
     <normaloff>:Icons/Pencil-icon.png</normaloff>:Icons/Pencil-icon.png</iconset>
   </property>
   <property name="text">
    <string>Edit </string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:Icons/Button-Close-icon.png</normaloff>:Icons/Button-Close-icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemove_Entry">
   <property name="icon">
    <iconset>
     <normaloff>:Icons/database-remove-icon.png</normaloff>:Icons/database-remove-icon.png</iconset>
   </property>
   <property name="text">
    <string>Remove </string>
   </property>
  </action>
  <action name="actionAbout_Developer">
   <property name="icon">
    <iconset>
     <normaloff>:Icons/Very-Basic-About-icon.png</normaloff>:Icons/Very-Basic-About-icon.png</iconset>
   </property>
   <property name="text">
    <string>About </string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>:Icons/QtProject-creator-icon.png</normaloff>:Icons/QtProject-creator-icon.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
